src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -716,6 +773,27 @@ AS
 
     SELECT @newWatermark
 
+    -- Insert metadata into custom tag tables
+    INSERT INTO dbo.CustomTagString
```

Comment:
what happens if this is a study level tag and you get duplicate in each instance. You can see it being handled for study table above. You need a upsert below.

Also add unit tests